lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -19,7 +31,11 @@ def define
           if response == 'yes'
             question :working_tax_credit?
           else
-            outcome :outcome_not_affected_no_housing_benefit
+            if chosen_cap == 'future'
```

Comment:
I think this would be much easier to understand for people not familiar with this Smart Answer if it was written as `if period_to_check == 'post-autumn-2016'` (i.e. both `chosen_cap` and the `'future'` and `'current'` options are renamed).